kernel/base/build.gradle

```diff
@@ -39,7 +39,9 @@ dependencies {
   compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.6'
   compile group: 'org.apache.commons', name: 'commons-text', version: '1.1'
   compile group: 'org.apache.maven.shared', name: 'maven-invoker', version: '3.0.0'
-  compile group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.22'
+  compile group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.25'
```

Comment:
You shouldn't have two logging implementations on the classpath. I would recommend switching this line from `slf4j-simple` to `slf4j-api`. That will allow you to use the slf4j logging API with the logback backend implementation